backend/users/views.py

```diff
@@ -1,3 +1,22 @@
 from django.shortcuts import render
+from .models import User
+from .serializers import UserSerializer, UserCreateSerializer
+from rest_framework import viewsets
+from rest_framework.response import Response
 
-# Create your views here.
+
+class UserViewSet(viewsets.ModelViewSet):
+    queryset = User.objects.all()
+    serializer_classes = {
+        'list': UserSerializer,
+        'get': UserSerializer,
+        'create': UserCreateSerializer,
+        'update': UserSerializer
+    }
```

Comment:
I found [this post](https://habr.com/en/post/306538/) on habr and was curious to try and see whether it could be useful for my task, but in the end it seemed like too much code, so I chose to override the `get_serializer_class` method instead, just like the official documentation suggests. Absolutely forgot to remove the `serializer_classes` part, though, it's not needed there.